### ISSUE TITLE
relative path for command not recognized with yaml config

### ISSUE THREAD

**Issue by roylez** [adnanh/webhook#268]
It works as expected if I change the config format to json.

```
2018/09/18 17:22:15 hooks file /etc/webhook.conf overwritten
2018/09/18 17:22:15 attempting to reload hooks from /etc/webhook.conf
2018/09/18 17:22:15 found 1 hook(s) in file
2018/09/18 17:22:15         loaded: redeploy
2018/09/18 17:22:20 [93e796] incoming HTTP request from x.x.x.x:36314
2018/09/18 17:22:20 [93e796] redeploy got matched
2018/09/18 17:22:20 [93e796] error parsing JSON payload EOF
2018/09/18 17:22:20 [93e796] redeploy hook triggered successfully
2018/09/18 17:22:20 200 | 285.371µs | tyo1.cloaked.me:6006 | POST /hooks/redeploy
2018/09/18 17:22:20 unable to locate command: './deploy.sh'
2018/09/18 17:23:28 hooks file /etc/webhook.conf overwritten
2018/09/18 17:23:28 attempting to reload hooks from /etc/webhook.conf
2018/09/18 17:23:28 found 1 hook(s) in file
2018/09/18 17:23:28         loaded: redeploy
2018/09/18 17:23:34 [55e156] incoming HTTP request from x.x.x.x:36318
2018/09/18 17:23:34 [55e156] redeploy got matched
2018/09/18 17:23:34 [55e156] error parsing JSON payload EOF
2018/09/18 17:23:34 [55e156] redeploy hook triggered successfully
2018/09/18 17:23:34 200 | 1.942987ms | tyo1.cloaked.me:6006 | POST /hooks/redeploy
2018/09/18 17:23:34 [55e156] error extracting command arguments: couldn't retrieve argument for {Source:payload Name:build_id EnvName: Base64Decode:false}
2018/09/18 17:23:34 [55e156] executing /root/ss/deploy.sh (/root/ss/deploy.sh) with arguments ["/root/ss/deploy.sh" ""] and environment [] using /root/ss as cwd                       
```

**Comment by adnanh** [comment]
Could you please post an example hooks.yaml file that can be used to reproduce this. Thank you :-)

**Comment by roylez** [comment]
Here is a minimal config file. Neither `deploy.sh` nor `./deploy.sh` works. Webook version is 2.6.8, it worked when with earlier versions.

``` yaml
---
- id: redeploy
  execute-command: "./deploy.sh"
  command-working-directory: /root/ss
```

**Comment by rafaelsimonmaia** [comment]
Most likely you have a permission issue with your file. Try something like chmod 517 deploy.sh and see if it works.

**Comment by roylez** [comment]
It is not permission issue. I am using absolute path now and it works.

Rafael Simon Maia <notifications@github.com> 于 2018年11月28日周三 06:55写道：

**Comment by moorereason** [comment]
Fixed since 2.6.10 with #327
